frontend/src/views/main/project/modals/AddTestToSuite.vue

```diff
@@ -10,8 +10,9 @@
           <v-card-text>
             <v-row>
               <v-col cols=12>
-                <ValidationProvider name="Tes suite" mode="eager" rules="required" v-slot="{ errors }">
-                  <v-select outlined label="Test suite" v-model="selectedSuite" :items="testSuites" :item-text="'name'" :item-value="'id'" dense hide-details></v-select>
+                <ValidationProvider name="Test suite" mode="eager" rules="required" v-slot="{ errors }">
+                  <v-select outlined label="Test suite" v-model="selectedSuite" :items="[...testSuites, { id: 0, name: '+ New test suite' }]" :item-text="'name'" :item-value="'id'" dense hide-details @change="checkIfCreateNew">
```

Comment:
It's cleaner to use slot to happen item: https://v2.vuetifyjs.com/en/components/selects/#append-and-prepend-item